File: Stacked_Grid_Client/Stacked_Grid_Client/Commands/ICommandHandler.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Stacked_Grid_Client.Commands
{
    public interface ICommandHandler<U>        
    {
        void handle(ICommand<U> command);

        
    }
}

File: Server/Server/src/server/Commands/ICommandHandler.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package server.Commands;

/**
 *
 * @author conno
 * @param <T>
 */
public interface ICommandHandler<T extends ICommand<?>> {

    void Handle(T command);
}
